File: GooglePay/src/main/java/com/braintreepayments/api/BraintreeGooglePayWalletConstants.java
package com.braintreepayments.api;

import com.google.android.gms.wallet.WalletConstants;

/**
 * Collection of constant values used by the Braintree SDK Google Payment module. Extends upon
 * com.google.android.gms.wallet.WalletConstants.
 */
class BraintreeGooglePayWalletConstants {

    /**
     * Card network Elo.
     */
    static final int CARD_NETWORK_ELO = WalletConstants.CARD_NETWORK_OTHER + 1;
}

File: GooglePay/src/main/java/com/braintreepayments/api/googlepay/BraintreeGooglePayWalletConstants.kt
package com.braintreepayments.api.googlepay

import com.google.android.gms.wallet.WalletConstants

/**
 * Collection of constant values used by the Braintree SDK Google Payment module. Extends upon
 * com.google.android.gms.wallet.WalletConstants.
 */
internal object BraintreeGooglePayWalletConstants {
    /**
     * Card network Elo.
     */
    const val CARD_NETWORK_ELO: Int = WalletConstants.CARD_NETWORK_OTHER + 1
}
